recipe/meta.yaml

```diff
@@ -23,12 +23,12 @@ requirements:
     - pkg-config  # [not win]
     - python
     - pip
-    - zeromq >=4.2.3,<4.3  # [not win]
-    - libsodium   # [not win]
+    - zeromq >=4.2.3,<4.3
```

Comment:
This is going to require zeromq 4.2.5 on Windows because of the .lib location fix, right?

Reply:
Righy